cluster-provision/gocli/cmd/run.go

```diff
@@ -526,25 +524,25 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 		}
 
 		if success {
-            // move the VM sound card to a vfio-pci driver to prepare for assignment
-            err = prepareDeviceForAssignment(cli, nodeContainer(prefix, nodeName), soundcardPCIID, "")
-            if err != nil {
-                return err
-            }
+			// move the VM sound card to a vfio-pci driver to prepare for assignment
```

Comment:
We should add go-formatting. :/ Can we keep whitespaces?

Reply:
Never mind. This was go-fmt changing things :+1:

Reply:
tab is the standard for go, let's try to add those checks at follow up.

Reply:
Definitely. :+1: